### PR TITLE
Update qcmetrics

### PR DESCRIPTION
**Pull request by lindaxiang**
Test cases:
- Prealnqc: 
```
nextflow run ./tests/modules/icgc-argo-workflows/payload/qcmetrics -entry test_payload_qcmetrics -c ./tests/config/nextflow.config -c ./tests/modules/icgc-argo-workflows/payload/qcmetrics/nextflow.prealn.config -profile docker
```
-dnaalnqc
```
nextflow run ./tests/modules/icgc-argo-workflows/payload/qcmetrics -entry test_payload_qcmetrics -c ./tests/config/nextflow.config -c ./tests/modules/icgc-argo-workflows/payload/qcmetrics/nextflow.aln.config -profile docker
```

### CODE REVIEW

**Review by edsu7 — COMMENTED** [review]
Is this line still used anywhere?
https://github.com/icgc-argo-workflows/argo-modules/blob/681bcc1563700f8322be1ba65305a85e169f0909/tests/config/test_data.config#L27

**Review by edsu7 — COMMENTED** [review]
Is this portion to handle 25k schema still needed?

https://github.com/icgc-argo-workflows/argo-modules/blob/681bcc1563700f8322be1ba65305a85e169f0909/modules/icgc-argo-workflows/payload/qcmetrics/resources/usr/bin/main.py#L282-L290.

Is it not accounted when we generate payloads for intermediate?

**Comment by lindaxiang** [comment]
> argo-modules/tests/config/test_data.config

Yes, this is needed for test case here: https://github.com/icgc-argo-workflows/argo-modules/blob/681bcc1563700f8322be1ba65305a85e169f0909/tests/modules/icgc-argo-workflows/prep/metrics/main.nf#L9
Unless the user provide their own params.qc_files

**Comment by lindaxiang** [comment]
> Is this portion to handle 25k schema still needed?
> 
> https://github.com/icgc-argo-workflows/argo-modules/blob/681bcc1563700f8322be1ba65305a85e169f0909/modules/icgc-argo-workflows/payload/qcmetrics/resources/usr/bin/main.py#L282-L290
> 
> .
> Is it not accounted when we generate payloads for intermediate?

This is the code to inherit the `info` field from input analysis to output analysis. Not directly related to ICGC25k cases.

File: tests/data/qa/875ef550-e536-4456-9ef5-50e5362456df.analysis.json [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  "experiment" : {
+    "experimental_strategy" : "WGS",
+    "platform" : "ILLUMINA",
+    "platform_model" : "HiSeq 2000",
+    "sequencing_center" : "EXT",
+    "sequencing_date" : "2014-12-12",
+    "submitter_sequencing_experiment_id" : "SubWf_exp_02_FASTQ_input"
+  },
+  "read_group_count" : 3,
+  "read_groups" : [
```
Comment (edsu7) [inline tests/data/qa/875ef550-e536-4456-9ef5-50e5362456df.analysis.json:138]: Double checking we're okay with qcmetrics.jsons not having `read_groups`?

**Review by edsu7 — APPROVED** [review]
tested in both docker and singularity. lgtm

**Pull request merged**